src/current/v23.2/physical-cluster-replication-overview.md

```diff
@@ -104,7 +104,7 @@ To connect to a cluster using the SQL shell:
     ~~~
 
 {{site.data.alerts.callout_info}}
-Physical cluster replication requires an {{ site.data.products.enterprise }} license on the primary and standby clusters. You must set {{ site.data.products.enterprise }} licenses from the system interface.
+Physical cluster replication requires an [{{ site.data.products.enterprise }} license]({% link {{ page.version.version }}/enterprise-licensing.md %}) on the primary and standby clusters. You must set {{ site.data.products.enterprise }} licenses from the system interface.
```

Comment:
nit: for `license` maybe use this more specific link: https://www.cockroachlabs.com/docs/v23.2/licensing-faqs#obtain-a-license